### PR TITLE
Access mgmt updates february

### PR DESCRIPTION
**Pull request by sterobin**
Updated access management content per latest changes in UI with predefined permission options.

Doc previews:
* [Managing access](http://file.rdu.redhat.com/~sterobin/MGDSTRM-access-mgmt/README.html)
* [Getting started > Setting permissions](http://file.rdu.redhat.com/~sterobin/MGDSTRM-getting-started/README.html#proc-setting-service-account-permissions_getting-started)

Jira: https://issues.redhat.com/browse/MGDSTRM-5849

### CODE REVIEW

**Comment by katheris** [comment]
I've reviewed the changes and they all make sense to me

File: access-mgmt/README.adoc [hunk 1]
```diff
@@ -238,20 +247,29 @@ h|Access type
...
+|`Topic`
+
+(For producing)
+|`Starts with` = `*`
 |`Allow`
-|`All`
+|`Write`, `Create`, `Describe`
 |===
+
+NOTE: Alternatively, you can click *Add permission* to individually create one `Topic` entry and one `Consumer group` entry, both with `Allow` access to `All` operations. This enables both consuming and producing for the topic in a single entry, and enables all permission for the consumer group in a single entry. But you must configure these entries individually without using the predefined permission options.
```
Comment (bhardesty) [inline access-mgmt/README.adoc:271]:
```suggestion
NOTE: Alternatively, you can click *Add permission* to individually create one `Topic` entry and one `Consumer group` entry, both with `Allow` access to `All` operations. This enables both consuming and producing for the topic in a single entry, and enables all permissions for the consumer group in a single entry. But you must configure these entries individually without using the predefined permission options.
```

File: getting-started/README.adoc [hunk 2]
```diff
@@ -215,20 +217,29 @@ h|Access type
...
+|`Topic`
+
+(For producing)
+|`Starts with` = `*`
 |`Allow`
-|`All`
+|`Write`, `Create`, `Describe`
 |===
+
+NOTE: Alternatively, you can click *Add permission* to individually create one `Topic` entry and one `Consumer group` entry, both with `Allow` access to `All` operations. This enables both consuming and producing for the topic in a single entry, and enables all permission for the consumer group in a single entry. But you must configure these entries individually without using the predefined permission options.
```
Comment (bhardesty) [inline getting-started/README.adoc:241]:
```suggestion
NOTE: Alternatively, you can click *Add permission* to individually create one `Topic` entry and one `Consumer group` entry, both with `Allow` access to `All` operations. This enables both consuming and producing for the topic in a single entry, and enables all permissions for the consumer group in a single entry. But you must configure these entries individually without using the predefined permission options.
```

**Review by bhardesty — APPROVED** [review]
Looks great, just a really minor suggestion.

File: access-mgmt/README.adoc (hunk 1 above)
Comment (sterobin) [reply access-mgmt/README.adoc]: Done on both. Thanks @bhardesty !

**Comment by mmusil** [comment]
Looks good to me. I wanted to check the UI while reading this but that does not work at the moment.

**Comment by sterobin** [comment]
Thanks @mmusil !

**Pull request merged**